src/webgpu/api/operation/adapter/requestDevice_limits.spec.ts

```diff
@@ -46,11 +47,13 @@ g.test('default_limits')
   .fn(async t => {
     const { limit } = t.params;
 
-    const adapter = await navigator.gpu.requestAdapter();
+    const gpu = getGPU();
+    const adapter = await gpu.requestAdapter();
     assert(adapter !== null);
 
-    const requiredLimits: Record<string, number> = {};
-    requiredLimits[limit] = DefaultLimits[limit];
+    if (adapter.limits) { return; }
```

Comment:
this check seems wrong?

Reply:
That was an accidental leftover from some previous debugging. I've got it removed locally, but I'm kind of waiting to see what happens with https://github.com/gpuweb/gpuweb/pull/2020 before pushing this any futher, because if that lands then I can solve some of my problems with this CL by updating the webgpu types and updating the defaults in the CTS.